### ISSUE TITLE
Crash on page without date

### ISSUE THREAD

**Issue by VorpalBlade** [dArignac/pelican-extended-sitemap#10]
Plugin causes pelican to crash if a page has no date:

Using the debug mode of pelican I get:
```
CRITICAL: AttributeError: 'Page' object has no attribute 'date'
Traceback (most recent call last):
  File "/home/user/virtualenv/pelican/bin/pelican", line 10, in <module>
    sys.exit(main())
  File "/home/user/virtualenv/pelican/lib/python3.7/site-packages/pelican/__init__.py", line 609, in main
    autoreload(watchers, args, old_static, reader_descs)
  File "/home/user/virtualenv/pelican/lib/python3.7/site-packages/pelican/__init__.py", line 513, in autoreload
    pelican.run()
  File "/home/user/virtualenv/pelican/lib/python3.7/site-packages/pelican/__init__.py", line 190, in run
    p.generate_output(writer)
  File "/home/user/virtualenv/pelican/lib/python3.7/site-packages/extended_sitemap/__init__.py", line 110, in generate_output
    pages_sorted = sorted(self.context.get('pages'), key=self.__get_date_key, reverse=True)
  File "/home/user/virtualenv/pelican/lib/python3.7/site-packages/extended_sitemap/__init__.py", line 205, in __get_date_key
    return getattr(obj, 'modified', None) or obj.date
AttributeError: 'Page' object has no attribute 'date'
```

Since I use pages for static things I have not put dates in them, and this seems to work except for with this plugin.

Software versions:
* pelican-extended-sitemap 1.0.5 (from PyPI)
* Pelican 4.0.1 (from PyPI)

**Comment by dArignac** [comment]
Hi @VorpalBlade,

thanks for the bug report.
It might be because I did not yet adjust to Pelican 4.x

**Comment by dArignac** [comment]
Hi @VorpalBlade, this is now fixed in [1.1.0](https://pypi.org/project/pelican-extended-sitemap/1.1.0/).
